packages/experimental-loader/test/projects/errors-integration/index.st.css

```diff
@@ -3,6 +3,6 @@
     -st-default: NotFound;
 }
 
-.root {
-    -st-extends: Unknown;
-}
\ No newline at end of file
+Unknown {
```

Comment:
why change?

Reply:
to match the required diagnostic severity in the test